network/policy/policy_windows.go

```diff
@@ -427,6 +441,33 @@ func GetHcnL4WFPProxyPolicy(policy Policy) (hcn.EndpointPolicy, error) {
 	return l4WfpEndpolicySetting, nil
 }
 
+// GetHcnLoopbackDSRPolicy policy is for pod to reach itself by cluster service IP.
```

Comment:
can we document it here that it's only supported on HNSV2 path which for containerd only and not supported for docker?

Reply:
yes its supported for hnsv2 but not for v1..i will document that